### DISCUSSION TITLE
Is this a Trojan or Rootkit? I installed that like a year ago.

### DISCUSSION THREAD

**Discussion by MrRaja23** [community/community#48254] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

https://github.com/Kammerflimmern/Twitch-TikTok-Youtube-Viewbot

This GitHub made me delete my discord and change all my passwords and more. I want some Security expert to look into this and figure out if this is a RootKit or normal Remote Access Trojan. I need to know if i have to replace my entire hardware or not.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Nielymmah ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
